src/vmm/src/config/builder.rs

```diff
@@ -0,0 +1,110 @@
+//! Config builder
+use std::convert::TryFrom;
+
+use super::{
+    BlockConfig, ConfigResult, ConversionError, KernelConfig, MemoryConfig, NetConfig, VMMConfig,
+    VcpuConfig,
+};
+
+/// Builder structure for VMMConfig
+#[derive(Debug)]
+pub struct Builder {
+    inner: ConfigResult<VMMConfig>,
```

Comment:
I find it a bit more readable without the `ConfigResult` type definition. When first reading the code, I thought that `ConfigResult` is a fancy type, and not just a wrapper over Result.

If you would like to keep the type, can you move it to this module instead?

Reply:
Ok Sure. I placed it inside `config/mod.rs`, because I thought it's generally going to be useful for all `config` related activities (like parser etc.), May be `ConfigResult` is a bad name :-)  For now will move to a simple `Result`.